CHANGELOG.md

```diff
@@ -1,23 +1,45 @@
-##  Next version
+##  2021-01-19 v0.7
 
-This release contains the following:
+Some useful features and fixes in this release, include:
```

Comment:
Can also consider adding (issue) or (pr) after for people who want more details. Or not, just an idea

Reply:
Done in release notes.